internal/pkg/client/cache/library_test.go

```diff
@@ -35,16 +48,84 @@ func TestLibrary(t *testing.T) {
 	}
 }
 
-// Will come back to after functionality exists
-// func TestLibraryImageExists(t *testing.T) {
-// 	tests := []struct {
-// 		name     string
-// 		env      string
-// 		expected string
-// 		file     []struct {
-// 			create bool
-// 			sum    string
-// 			name   string
-// 		}
-// 	}{}
-// }
+func TestLibraryImage(t *testing.T) {
+	// LibraryImage just return a string and there is no definition of what
+	// could be a bad string.
+	tests := []struct {
+		name     string
+		sum      string
+		path     string
+		expected string
+	}{
+		{
+			name:     "General case",
+			sum:      validSHASum,
+			path:     validPath,
+			expected: filepath.Join(Library(), "", ""),
```

Comment:
Does `filepath.Join()` need to be called here? It seems like it is not necessary?...

Reply:
Thanks and fixed.